bluemira/plasma_physics/reactions.py

```diff
@@ -174,3 +178,302 @@ def r_D_burn_DT(p_fus):  # noqa :N802
         {17.58 [MeV]eV[J]N_{A}[1/mol]} [g/s]
     """
     return n_DT_reactions(p_fus) * D_MOLAR_MASS / N_AVOGADRO
+
+
+REACTIONS = {
+    "D-T": 0,  # D + T --> 4He + n reaction
+    "D-D": -1,  # D + D --> 0.5 D-D1 + 0.5 D-D2
+    "D-D1": 1,  # D + D --> 3He + n reaction [50 %]
+    "D-D2": 2,  # D + D --> T + p reaction [50 %]
+    "D-He3": 3,  # D + 3He --> 4He + p reaction
+}
```

Comment:
This feels very enum-y

```suggestion
class Reactions(Enum):
    D_T =  auto(),  # D + T --> 4He + n reaction
    D_D = auto(),  # D + D --> 0.5 D-D1 + 0.5 D-D2
    D_D1 = auto(),  # D + D --> 3He + n reaction [50 %]
    D_D2 =  auto(),  # D + D --> T + p reaction [50 %]
    D_He3 = auto(),  # D + 3He --> 4He + p reaction

```